src/middlewared/middlewared/plugins/service_/services/pseudo/misc.py

```diff
@@ -247,12 +247,7 @@ class DSCacheService(PseudoServiceBase):
     name = "dscache"
 
     async def start(self):
-        ldap_enabled = (await self.middleware.call('ldap.config'))['enable']
-        if ldap_enabled:
-            await systemd_unit("nscd", "restart")
-        else:
-            await systemd_unit("nscd", "stop")
-
+        await systemd_unit("nscd", "start")
```

Comment:
Shouldn'it it be just always started now?

Reply:
Good point.